Cargo.toml

```diff
@@ -27,7 +27,7 @@ features = [
 ]
 
 [features]
-default = ["default-tls", "charset", "http2", "macos-system-configuration"]
+default = ["default-tls", "charset", "http2", "macos-system-configuration", "hickory-dns"]
```

Comment:
I just enabled this as a default so that `cargo build` works by default locally for us